src/jit/importer.cpp

```diff
@@ -1984,6 +1984,12 @@ void                Compiler::impSpillStackEnsure(bool spillLeaves)
     {
         GenTreePtr      tree = verCurrentState.esStack[level].val;
 
+        // Don't spill explicit tail call result. A tail-prefixed call is converted to a jump.
```

Comment:
I would suggest to modify the above comment as follows:

_Don't spill explicit tail call result, because a tail call never returns to its caller._